package.json

```diff
@@ -45,6 +48,7 @@
     "jsdoc-to-markdown": "^4.0.1",
     "keythereum": "^1.0.4",
     "node-yaml": "^3.1.1",
+    "pluck": "0.0.4",
```

Comment:
I would use `lodash.get` instead. It's possible to install only the `get` function without having to install all of lodash.
https://www.npmjs.com/package/lodash.get

`pluck` has not received any code updates the last 5 years, has 372 weekly downloads (vs 1,7 million weekly downloads of lodash.get) and has no link to any repos to inspect the code.

Reply:
True, as long as we dont end up pulling a monster sized lodash / underscore / jquery lib. We should strive to keep dependencies as lean and specific as possible.